src/subgraph.js

```diff
@@ -111,6 +117,25 @@ module.exports = class Subgraph {
     )
   }
 
+  static validateMutationResolvers(manifest, { resolveFile }) {
+    if (manifest.get('mutations')) {
+      const resolversFile = manifest.getIn(['mutations', 'resolvers', 'file'])
+      const schemaFile = manifest.getIn(['mutations', 'schema', 'file'])
+      const errorMsg = validation.validateMutationResolvers(resolversFile, schemaFile, { resolveFile })
+
+      if (errorMsg) {
+        return immutable.fromJS([
+          {
+            path: ['mutations', 'resolvers', 'file'],
+            message: errorMsg
+          }
+        ])
+      }
+    }
```

Comment:
We try to collect as many errors as possible at once to reduce the amount of trial-and-error users have to go through. If `validateMutationResolvers` can return more than one error, we should have it return a list of errors and not just an error message.